kubernetes/utils.go

```diff
@@ -176,12 +176,11 @@ func getK8Config(ctx context.Context, d *plugin.QueryData) (clientcmd.ClientConf
 	kubernetesConfig := GetConfig(d.Connection)
 
 	// Set default loader and overriding rules
-	loader := &clientcmd.ClientConfigLoadingRules{}
+	loader := clientcmd.NewDefaultClientConfigLoadingRules()
```

Comment:
@mrkwtz @bigdatasourav - I'm not familiar with the default loader, but would either of you be able to please explain what environment variables this plugin now looks at and in which order? I think we should also update all of the references to file locations in `config/kuberenetes.spc`, `docs/index.md`, and the `README.md` (if there any here), so users can better understand what this plugin supports.

Reply:
Hey @mrkwtz, Could you please update the reference files mentioned above with how the KUBECONFIG env variable will work and in which order?